tensorflow/core/util/util.cc

```diff
@@ -121,19 +122,26 @@ string SliceDebugString(const TensorShape& shape, const int64 flat) {
 }
 
 #ifdef INTEL_MKL
-bool DisableMKL() {
-  enum MklStatus { MKL_DEFAULT = 0, MKL_ON = 1, MKL_OFF = 2 };
-  static MklStatus status = MKL_DEFAULT;
-  if (status == MKL_DEFAULT) {
-    char* tf_disable_mkl = getenv("TF_DISABLE_MKL");
-    if ((tf_disable_mkl != NULL) && (std::stoi(tf_disable_mkl) == 1)) {
-      VLOG(2) << "TF-MKL: Disabling MKL";
-      status = MKL_OFF;
-    } else {
-      status = MKL_ON;
-    }
-  }
-  return status == MKL_OFF ? true : false;
+bool IsMKLEnabled() {
```

Comment:
Can we move `#ifdef INTEL_MKL` inside this function? E.g.,
```c++
bool IsMKLEnabled() {
#ifndef INTEL_MKL
  return false;
#else 
  ...
#endif  // !INTEL_MKL
}
```

I'm also fine if you want to keep the whole function inside `#ifdef INTEL_MKL` for now. (E.g., `IsMKLEnabled` won't exist if `-DINTEL_MKL` is not provided.)

Reply:
Actually your suggestion was my original implementation.  I will make the change back even though IsMKLEnabled() 
always exists. This will make TF code much cleanly -- currently wherever IsMKLEnabled() is invoked, we have to add 
"#ifdef INTEL_MKL" protection :). 

For this PR, I will change IsMKLEnable method. We will refactor its usage places later.

Reply:
Sounds good. Thank you! :)